sample/scopes/build.gradle

```diff
@@ -2,4 +2,5 @@ apply plugin: 'org.jetbrains.kotlin.jvm'
 
 dependencies {
   api deps.kotlin.stdlib
+  api "javax.inject:javax.inject:1"
```

Comment:
Ah sorry, I missed that in the first review. In this repository we list all dependencies in the `dependencies.gradle` file. You should do the same for this dependency and then use `api deps.inject` here.

Reply:
Sounds good, I'll make that update!